www/javascripts/missions.js

```diff
@@ -80,13 +100,7 @@
 					playerId: '=',
 					missionId: '='
 				},
-				controller: ['Errors', 'Players', function (Errors, Players) {
-					$playerMissionCtrl = this;
-					Players.getMissionStatus(this.playerId, this.missionId,
-						function(data) {
-							$playerMissionCtrl.missionStatus = data;
-						}, Errors.handleError);
-				}],
+				controller: 'PlayerMissionCtrl',
```

Comment:
wat? I dont know if I really want to understand this part, but the new code seems simpler.